GEOSldas_App/ldas_setup

```diff
@@ -65,77 +66,93 @@ class LDASsetup:
+        self.verbose            = cmdLineArgs['verbose']
+
+        # command line args for coupled land-atm DAS (see "help" strings in parseCmdLine() for details)
+        self.nymdb              = cmdLineArgs['nymdb']
+        self.nhmsb              = cmdLineArgs['nhmsb']
+        self.agcm_res           = cmdLineArgs['agcm_res']
+        self.bcs_version        = cmdLineArgs['bcs_version']
+        self.rstloc             = cmdLineArgs['rstloc']
+        self.varwindow          = cmdLineArgs['varwindow']
+        self.LDAS_nml_path      = cmdLineArgs['LDAS_nml_path']
+        self.mwrtm_path         = cmdLineArgs['mwrtm_path']
+
+        # obsolete command line args
+        self.runmodel           = cmdLineArgs['runmodel']
         if self.runmodel :
           print('\n The option "--runmodel" is out of date, not necessary anymore. \n')
-        self.daysperjob = cmdLineArgs['daysperjob']
-        self.monthsperjob = cmdLineArgs['monthsperjob']
-        self.rqdExeInp = OrderedDict()
-        self.rqdRmInp = OrderedDict()
-        self.optRmInp = OrderedDict()
-        self.rundir    = None
-        self.blddir   = None
-        self.blddirLn = None
-        self.outdir   = None
-        self.out_path   = None
-        self.inpdir   = None
-        self.exefyl = None
-        self.isZoomIn = False
-        self.catch = ''
-        self.has_mwrtm = False
-        self.has_vegopacity = False
-        self.assim = False
+
+        self.daysperjob         = cmdLineArgs['daysperjob']
+        self.monthsperjob       = cmdLineArgs['monthsperjob']
+
+        self.rqdExeInp          = OrderedDict()
+        self.rqdRmInp           = OrderedDict()
+        self.optRmInp           = OrderedDict()
+        self.rundir             = None
+        self.blddir             = None
+        self.blddirLn           = None
+        self.outdir             = None
+        self.out_path           = None
+        self.inpdir             = None
+        self.exefyl             = None
+        self.isZoomIn           = False
+        self.catch              = ''
+        self.has_mwrtm          = False
+        self.has_vegopacity     = False
+        self.assim              = False
         self.has_landassim_seed = False
-        self.has_geos_pert = False
-        self.nSegments = 1
-        self.perturb = 0
-        self.first_ens_id = 0
-        self.ladas_coupling = 0
-        self.in_rstfile = None
-        self.in_tilefile = 'None' # default string
-        self.ens_id_width = 6     # _eXXXX
-        self.bcs_land = ''
-        self.bcs_geom = ''
-        self.bcs_landshared = ''
-        self.tile_types = ''
-        self.with_land = False
-        self.with_landice = False
+        self.has_geos_pert      = False
+        self.nSegments          = 1
+        self.perturb            = 0
+        self.first_ens_id       = 0
+        self.in_rstfile         = None
+        self.in_tilefile        = 'None'                 # default string
```

Comment:
@weiyuan-jiang , @saraqzhang : Why do we use None (without quotation marks) for self.in_rstfile and 'None' (with quotation marks) for self.in_tilefile?  Can't we make this consistent?

Reply:
'None' is a str. If you don't provide default in the command line argument, it will be None ( a type in python). For example, here https://github.com/GEOS-ESM/GEOSldas_GridComp/blob/ae2717e9bddc2ea11ba9fcc96da929b8abfdb23a/GEOSldas_App/ldas_setup#L1915
 The test would be like
if cmdLineArgs['account'] is not None: